jekyll/_cci2/configuration-reference.md

```diff
@@ -539,6 +539,48 @@ The directory layout should match the [classic CircleCI test metadata directory
     path: /tmp/test-results
 ```
 
+#### **`persist_to_workspace`**
+
+Special step used to persist a temporary file to be used by another job in the workflow. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+paths | Y | String | Directory containing temporary files
+{: class="table table-striped"}
+
+##### _Example_
+
+``` YAML
+- persist_to_workspace:
+    paths: /tmp/file-name
+```
+
+#### **`attach__workspace`**
+
+Special step used to attach a temporary file generated by another job in the workflow. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+at | Y | String | Directory containing temporary files
+{: class="table table-striped"}
+
+##### _Example_
+
+``` YAML
+- attach_workspace:
+    at: /tmp/file-name
+```
+
+Note the following distinctions between Artifacts, Workspaces, and Caches:
+
+| Type      | lifetime        | Use                      | Example |
+|-----------|-----------------|------------------------------------|---------
+| Artifacts | Months to years | Preserve long-term artifacts. |  Linked list in the Artifacts tab of the Build details under the `tmp/circle-artifacts.<hash>/container`   or similar directory.   |
+| Workspaces | Duration of workflow        | Move temporary files around for use by jobs run later in the workflow. | The `persist_to_workspace: <directory>` key under jobs, refer to the file with `attach_workspace: at: <directory>`.    |
```

Comment:
Should this content change? I can't tell from your comment if you want the words to reflect this.